helm/flink-kubernetes-operator/templates/rbac.yaml

```diff
@@ -21,23 +21,14 @@ RBAC rules used to create the operator (cluster)role based on the scope
 */}}
 {{- define "flink-operator.rbacRules" }}
 rules:
-  - apiGroups:
-      - flink-operator
-    resources:
-      - "*"
-    verbs:
-      - "*"
   - apiGroups:
       - ""
     resources:
       - pods
       - services
-      - endpoints
-      - persistentvolumeclaims
```

Comment:
The helm chart functionality can be extended with kustomize templates, and PVCs could be a useful to mount plugins and what not for the Operator. Although I'm generally in favour of least privilege principle, I would probably won't remove this one. WDYT?

Reply:
@morhidi Even if we are adding PVC to operator, operator is not creating/managing PVC. PVC is created outside of operator as I understood. So mounting a persistentVolumeClaim to operator deployment doesn't require this RBAC. `e2e-tests/data/sessionjob-cr.yaml` is a good example for this and that test is passed without the RBAC. Please correct me if I'm wrong.

Reply:
Good point, can be removed then